4788/src/main/cpp/Robot.cpp

```diff
@@ -3,60 +3,81 @@
 using namespace frc;
 using namespace wml;
 
+using hand = frc::XboxController::JoystickHand;
+
 double currentTimeStamp;
 double lastTimeStamp;
-double dt;
+double dt; //stands for delta time 
+
+//add other variables here
+double SparkSpeed;
+double TalonSpeed;
 
-// Robot Logiccd
+// Robot Logic
 void Robot::RobotInit() {
-	// Init the controllers
-	ControlMap::InitsmartControllerGroup(robotMap.contGroup);
+	
+	dt = frc::Timer::GetFPGATimestamp() - lastTimeStamp;
+	lastTimeStamp = frc::Timer::GetFPGATimestamp();
 
-	// Create wml drivetrain
-	drivetrain = new Drivetrain(robotMap.driveSystem.drivetrainConfig, robotMap.driveSystem.gainsVelocity);
+	//init controllers 
+	xbox = new frc::XboxController(0);
 
-	
-	// Zero Encoders
-	robotMap.driveSystem.drivetrain.GetConfig().leftDrive.encoder->ZeroEncoder();
-	robotMap.driveSystem.drivetrain.GetConfig().rightDrive.encoder->ZeroEncoder();
+	//Motor examples 
+	SparkMotor[0] = new frc::Spark(0);
+	TalonMotor = new wml::TalonSrx(1);
 
-	// Strategy controllers (Set default strategy for drivetrain to be Manual)
-	drivetrain->SetDefault(std::make_shared<DrivetrainManual>("Drivetrain Manual", *drivetrain, robotMap.contGroup));
-	drivetrain->StartLoop(100);
+	SparkMotor[0]->SetInverted(true);
+	TalonMotor->SetInverted(false);
 
-	// Inverts one side of our drivetrain
-	drivetrain->GetConfig().rightDrive.transmission->SetInverted(true);
-	drivetrain->GetConfig().leftDrive.transmission->SetInverted(false);
+	//registers the pressure sensor as a sensor to be logged to Network Tables
+	NTProvider::Register(&pressureSensor); 
 
-	// Register our systems to be called via strategy
-	StrategyController::Register(drivetrain);
-	NTProvider::Register(drivetrain);
 }
 
 void Robot::RobotPeriodic() {
-	currentTimeStamp = Timer::GetFPGATimestamp();
-	dt = currentTimeStamp - lastTimeStamp;
-
-	StrategyController::Update(dt);
-	NTProvider::Update();
-
-	lastTimeStamp = currentTimeStamp;
+	if (pressureSensor.GetScaled() < 80) {
+		compressor.SetTarget(wml::actuators::BinaryActuatorState::kForward); // turn the compressor on 
+	}
 }
 
 // Dissabled Robot Logic
-void Robot::DisabledInit() {}
+void Robot::DisabledInit() {
+	InterruptAll(true);
+}
 void Robot::DisabledPeriodic() {}
 
 // Auto Robot Logic
 void Robot::AutonomousInit() {}
 void Robot::AutonomousPeriodic() {}
 
 // Manual Robot Logic
-void Robot::TeleopInit() {
-	Schedule(drivetrain->GetDefaultStrategy(), true); // Use manual strategy
+void Robot::TeleopInit() {}
+void Robot::TeleopPeriodic() {
+
+	//motor examples
+	SparkSpeed = xbox->GetY(hand::kLeftHand);
+	SparkMotor[0]->Set(SparkSpeed);
+
+	TalonSpeed = xbox->GetTriggerAxis(hand::kRightHand);
+	if (TalonSpeed >= 0.1) { //acounts for the deadzone
```

Comment:
:(

`talonSpeed = xbox->GetTriggerAxis(hand::kRightHand) > deadzone ? xbox->GetTriggerAxis(hand::kRightHand) : 0;
_talonMotor->Set(talonSpeed);`

Reply:
preferably it would be 

`talonSpeed = kRightTrigger > deadzone ? kRightTrigger : 0;
_talonMotor->Set(talonSpeed)`